crates/types/src/services/txpool.rs

```diff
@@ -284,7 +284,7 @@ pub fn from_executor_to_status(
 #[non_exhaustive]
 pub enum Error {
     #[error("Gas price not found for block height {0}")]
-    GasPriceNotFound(BlockHeight),
+    GasPriceNotFound(String),
```

Comment:
The error message looks wrong when given non-`BlockHeight` values as you do in `crates/services/txpool/src/txpool.rs`.

Reply:
Consider introducing new error types for different errors as well, maybe `"Failed to calculated max block bytes"` should be a separate variant, for instance.